pkg/fatfs/fatfs_vfs/fatfs_vfs.c

```diff
@@ -0,0 +1,443 @@
+/*
+ * Copyright (C) 2017 HAW-Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     fs
+ * @{
+ *
+ * @file
+ * @brief       FatFs wrapper for vfs
+ *
+ * @author      Michel Rottleuthner <michel.rottleuthner@haw-hamburg.de>
+ *
+ * @}
+ */
+
+
+#include <fcntl.h>
+#define __LINUX_ERRNO_EXTENSIONS__
```

Comment:
just re-tested it: it actually isn't needed anymore. I initiallly added this because my old buildsystem complained about those missing errno extensions. I can't reproduce it anymore -> removed!